src/transaction.ts

```diff
@@ -163,22 +163,22 @@ export class Snapshot extends EventEmitter {
    * @type {?(string|Buffer)}
    */
   /**
-   * Whether or not the transaction has been ended. If true, no further requests
-   * should be made and the transaction should be discarded.
+   * Whether or not the transaction has ended. If true, no further requests
+   * are made and the transaction is discarded.
```

Comment:
I think this should be reverted, we're basically telling the user if they see this condition that they should discard the transaction object. Transactions have to be manually ended, which is why we say `should`.

Reply:
Can we say, "If true, make no further requests, and discard the transaction." ? This makes it in the active voice.

Reply:
SGTM!